core/src/main/java/hudson/tasks/BatchFile.java

```diff
@@ -101,28 +102,21 @@ public String getDisplayName() {
          * Performs on-the-fly validation of the errorlevel.
          */
         @Restricted(DoNotUse.class)
+        @SuppressRestrictedWarnings(UnstableAwareCommandInterpreter.class)
```

Comment:
Not really sure we want to use such tooling in the core.

Reply:
Is there some other way to make everyone happy?

Reply:
#4077